package/MDAnalysis/auxiliary/base.py

```diff
@@ -0,0 +1,447 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+"""
```

Comment:
Going to copy across something from the other PR here, replace analysis with auxiliary.  You will need to add an auxiliary directory and your own `auxiliary_modules.rst` file.

So for a documentation page to get built you also need to add a rst stub in package/doc/sphinx/source/documentation_pages/analysis (look at others in that dir) and a link to the page in documentation_pages/analysis_modules.rst

Each rst file becomes a page at docs.mdanalysis.org, and it's the automodule thing in the rst that basically just points to this source here for content.